pageserve/test_routes.py

```diff
@@ -0,0 +1,68 @@
+"""Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+from unittest.mock import patch
+from flask import Flask
+from flask_testing import TestCase
+import app
+
+EXAMPLE_USER = "app_user"
+EXAMPLE_POSTS = ['example', 'posts', 'list']
+EXAMPLE_EVENTS = ['example', 'events', 'list']
+
+
+class TestRoutes(TestCase):
```

Comment:
Have you thought of splitting the different endpoints into their own classes? I think that makes files like this (which will end up with a whole lot of tests) cleaner to navigate. Or if you think there aren't enough tests for each of these endpoints to have their own test class, alternatively you could make this `class TestGETRoutes` or similar.

Reply:
Renamed in 1bf1c5d